geany.appdata.xml.in

```diff
@@ -0,0 +1,43 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<component>
+  <id type="desktop">geany.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <_name>Geany IDE</_name>
+  <project_license>GPL-2.0+</project_license>
+  <_summary>A fast and lightweight IDE using GTK+</_summary>
+  <url type="homepage">http://geany.org/</url>
+  <description>
+    <_p>Geany is a small and lightweight integrated development environment.
```

Comment:
This line just repeats the summary, and its said again in the next line so I would drop it.